ui/src/variables/utils/hydrateVars.ts

```diff
@@ -35,25 +35,26 @@ interface HydrateVarsOptions {
 
 export const findDependentVariables = (
   variable: Variable,
-  allVariables: Variable[]
+  varGraph: VariableNode[]
 ): Variable[] => {
-  const query: string = variable.arguments.values.query
-  const dependencies = allVariables.filter(maybeChild =>
-    query.includes(`${OPTION_NAME}.${maybeChild.name}`)
-  )
-
-  return dependencies
+  const node = varGraph.find(n => n.variable.id === variable.id)
```

Comment:
Do you think you could move the export specific stuff to a separate utility module? `findDependentVariables` and `exportVariables`

Just because the `hydrateVars` module is so gnarly as is, I want to keep it as focused as possible

Reply:
was thinking that before that last commit. Will do.